daemons/based/pacemaker-based.c

```diff
@@ -399,9 +399,11 @@ cib_init(void)
     }
 
     if (!stand_alone) {
+#if SUPPORT_COROSYNC
         if (is_corosync_cluster()) {
             crm_set_status_callback(&cib_peer_update_callback);
         }
+#endif // SUPPORT_COROSYNC
```

Comment:
Nothing in cib_peer_update_callback() is specific to Corosync, so I'm thinking let's just drop the check. The only other possibility currently is standalone mode (which we never actually use), but even then the callback would simply never be called.

Reply:
Yeah there's `crm_proc_cpg` but that's not guarded, it's just an enum that we define. Sounds good.